core/repository/airdropHistory/index.ts

```diff
@@ -0,0 +1,21 @@
+import AirdropHistoryRepository from 'core/repository/airdropHistory/AirdropHistoryRepository';
+import AirdropHistoryFakeRepository from 'core/repository/airdropHistory/AirdropHistoryFakeRepository';
+import {
+    AirdropHistory, AirdropDailyHistoryQuery, AirdropHistoryQuery
+} from 'core/repository/airdropHistory/interfaces';
+import config from 'shared/config';
+
+export {
+    AirdropHistoryQuery,
+    AirdropHistory,
+    AirdropDailyHistoryQuery
+};
+
+const gerAirdropHistoryRepository = () => {
+    if (config.CORE.IS_REFERRED_USERS_ENABLED) {
+        return new AirdropHistoryRepository();
+    }
+    return new AirdropHistoryFakeRepository();
```

Comment:
What about returning an error when IS_REFERRED_USERS_ENABLED is disabled?

Reply:
The idea is not to change the client code when disabled